docs/data/base/guides/working-with-tailwind-css/PlayerFinal.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export default function PlayerFinal() {
+  return (
+    <iframe
+      title="codesandbox"
+      src="https://codesandbox.io/embed/mui-base-with-tailwind-css-42z1n8?hidenavigation=1&fontsize=14&view=preview"
+      style={{
+        width: '100%',
+        height: 350,
```

Comment:
Can we make it a bit taller? Both PlayerFinal and PlayerInteractive show vertical scrollbars to me (Edge on Windows 10, 150% pixel density)

Reply:
Maybe we could increase https://github.com/mui/material-ui/pull/33100#discussion_r929066849 too? Maybe it's fine to have a max height, but with the current height, it feels too small to be able to navigate the code when it overflows.

Reply:
Yep, makes sense will update